demo/index.tsx

```diff
@@ -1,13 +1,18 @@
 import React from 'react';
```

Comment:
What is this file compared to demo/src/index.tsx?

Reply:
It's old I think. We can [delete it](https://github.com/votemike/react-accordion/pull/6/files).